cmd/tools/cli/apply.go

```diff
@@ -11,21 +11,45 @@ import (
 var applyCmd = &cobra.Command{
 	Use:   "apply <plugin_name>",
 	Short: "Apply the plugin",
-	Args:  cobra.ExactArgs(1),
+	Args:  cobra.ExactArgs(2),
 	RunE: func(cmd *cobra.Command, args []string) error {
 		config := newConfig()
-		pluginName := args[0]
-		pluginSpecList, err := plugins.PluginYamls(pluginRepoURI, pluginName)
-		if err != nil {
-			return err
-		}
-		for _, pluginSpec := range pluginSpecList {
-			fmt.Printf("Plugin YAML content: %s, name: %s \n", pluginSpec.Yaml, pluginSpec.PluginName)
-			err = kctl.Apply(config, pluginSpec, nil)
-			if err != nil {
-				return err
+		commandType := args[0]
+		destination := args[1]
+
+		switch commandType {
```

Comment:
You can use cobra flag parsing to make this easier. The applyCmd can be returned from a function after applying a flag like
```go
	applyCmd.PersistentFlags().BoolVarP(&isGroup, "group", "g", false, "Deploy a plugin group")
```
The value would be set on a boolean var called: isGroup.

eg:
```go
var isGroup bool
func newApplyCommand() *cobra.Command {
	var applyCmd = &cobra.Command{
		Use:   "apply <plugin_name>",
		Short: "Apply the plugin",
		Args:  cobra.ExactArgs(1),
		RunE: func(cmd *cobra.Command, args []string) error {
			config := newConfig()
			pluginName := args[0]
			if isGroup {
				//Do something
			}
			....
			return nil
		},
	}
	applyCmd.PersistentFlags().BoolVarP(&isGroup, "group", "g", false, "Deploy a plugin group")
	return applyCmd
}